auth0-actions/allow-github-organisations-and-map-saml.js

```diff
@@ -52,7 +52,8 @@ exports.onExecutePostLogin = async (event, api) => {
         // Set SAML attribute for the user's GitHub team memberships
         const userTeamsResponse = await octokit.request('GET /user/teams').catch(error => api.access.deny(`Error retrieving teams from GitHub: ${error}`))
         const userTeamSlugs = userTeamsResponse.data.map(team => team.slug)
-        api.samlResponse.setAttribute('https://aws.amazon.com/SAML/Attributes/PrincipalTag:github_team', userTeamSlugs.join(','))
+        console.log("Joined the following teams:" `${userTeamSlugs.join(',')}`)
```

Comment:
Logline needs updating to 👇 

```suggestion
        console.log(`Joined the following teams: ${userTeamSlugs.join(',')}`)
```

Interpreter is getting confused about two separate strings being passed to the join function 🙈 

Will also need to update the test on line 64 to 👇 

```suggestion
             ['https://aws.amazon.com/SAML/Attributes/AccessControl:github_team', 'test-team-1,test-team-2'],
```

To reflect the new changes 🧪 🪞 😁